### DISCUSSION TITLE
Fork bei MacJoom

### DISCUSSION THREAD

**Discussion by MacJoom** [alexandreelise/j4x-api-collection#50] [Show and tell]
View my fork of the collection here: https://www.postman.com/macjoom/workspace/joomla/collection/20976681-4f03f3da-2199-49ac-b850-077350315dd8?action=share&creator=20976681

**Comment by alexandreelise** [comment]
Hi @MacJoom Here is the tooling proposed by the postman team to convert a postman collection to OpenAPI format: https://www.postman.com/postman/workspace/postman-open-technologies-convert-postman-collections-into-openapi/overview
